CMakeLists.txt

```diff
@@ -10,6 +13,17 @@ include(meshioChecks)
 option(MeshIO_BUILD_DOCS "Build MeshIO Documentation" OFF)
 option(MeshIO_BUILD_EXAMPLES "Build MeshIO Examples" OFF)
 option(MeshIO_BUILD_COVERAGE "Generate MeshIO coverage report" OFF)
+option(MeshIO_BUILD_TESTS "Build unit tests" OFF)
+
+if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
+  # Ensure -std=c++xx instead of -std=g++xx
+  set(CMAKE_CXX_EXTENSIONS OFF)
```

Comment:
This is better set as per target property instead of using global variables.

Reply:
Can't be set due to the property CMAKE_CXX_EXTENSIONS is not being whitelisted (https://gitlab.kitware.com/cmake/cmake/issues/17183). Also as noted by Brad King in the thread, it looks like this property does not have effect on INTERFACE libraries.

Hence I totally removed this.